src/gemm_v2.cpp

```diff
@@ -1267,15 +1286,16 @@ GemmDescriptor CreateGemmDescriptorConvFwd(const TensorDescriptor& wDesc,
     int n = std::accumulate(out_spatial.begin(), out_spatial.end(), 1, std::multiplies<int>());
     int k =
         in_c * std::accumulate(wei_spatial.begin(), wei_spatial.end(), 1, std::multiplies<int>());
-    int lda               = k;
-    int ldb               = wDesc.GetType() == miopenInt8 ? k : n;
-    int ldc               = n;
-    int batch_count       = 1;
-    long long int strideA = 0;
-    long long int strideB = 0;
-    long long int strideC = 0;
-    float alpha           = 1.;
-    float beta            = 0.;
+    int lda                  = k;
+    int ldb                  = wDesc.GetType() == miopenInt8 ? k : n;
+    int ldc                  = n;
+    int batch_count          = 1;
+    long long int strideA    = 0;
+    long long int strideB    = 0;
+    long long int strideC    = 0;
+    float alpha              = 1.;
+    float beta               = 0.;
+    const auto gfx90aaltimpl = problem.IsGfx90aFp16altRequired();
```

Comment:
[Note] `IsGfx90aFp16altRequired()` in the problem description is intended for use only in Solver that do not support ALT kernels, for disabling them.